src/cf-vars.less

```diff
@@ -44,15 +47,18 @@
     - cf-core
 */
 
-// Sizing variables 
+// Sizing variables
 
 @base-font-size-px:             16px;
 @base-line-height-px:           22px;
 @base-line-height:              unit(@base-line-height-px / @base-font-size-px);
 @mobile-max:                    599px;
 @tablet-min:                    600px;
+@tablet-max:                    800px;
+@desktop-min:                   801px;
+@wall-min:                      1200px;
```

Comment:
It's the breakpoint point for the "wall" or edges of the layout. I'm not attached to the name of the breakpoint if there's another suggestion. :smile: